dist.cmake

```diff
@@ -30,15 +30,21 @@ MACRO(_SETUP_PROJECT_DIST)
     FIND_PROGRAM(TAR tar)
     FIND_PROGRAM(GPG gpg)
 
+    IF(APPLE)
+      SET(GIT_ARCHIVE_ALL ${PROJECT_SOURCE_DIR}/cmake/git-archive-all.py)
+    ELSE(APPLE)
+      SET(GIT_ARCHIVE_ALL ${PROJECT_SOURCE_DIR}/cmake/git-archive-all.sh)
+    ENDIF(APPLE)
```

Comment:
Any reason not to use the Python one in both cases? (I'm guessing porting the shell version to OSX is more annoying that it's worth...)